multiqc/modules/picard/RnaSeqMetrics.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+
+""" MultiQC submodule to parse output from Picard RnaSeqMetrics """
+
+from collections import OrderedDict, defaultdict
+import logging
+import os
+import re
+
+from multiqc import config, plots
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+DESC = {
```

Comment:
Yep, I intend to copy+paste the metric names and descriptions from the
Picard website for RnaSeq. I left it there as a placeholder for myself...

On Thu, Jul 28, 2016 at 10:34 AM, Phil Ewels <notifications@github.com>
wrote:

> In multiqc/modules/picard/RnaSeqMetrics.py
> <https://github.com/ewels/MultiQC/pull/274#discussion_r72592021>:
>
> > @@ -0,0 +1,170 @@
> > +#!/usr/bin/env python
> > +
> > +""" MultiQC submodule to parse output from Picard RnaSeqMetrics """
> > +
> > +from collections import OrderedDict, defaultdict
> > +import logging
> > +import os
> > +import re
> > +
> > +from multiqc import config, plots
> > +
> > +# Initialise the logger
> > +log = logging.getLogger(__name__)
> > +
> > +DESC = {
>
> I think this is a placeholder to hold the long descriptions for each
> statistic, copied from the Picard website.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ewels/MultiQC/pull/274/files/0316e43dce18a097db87489f8250d777902604a1#r72592021>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJpN3WeKMxJbpHFO1RiYEfmLKdC9xOZks5qaHe9gaJpZM4JWEvx>
> .
>